content/docs/rosa/egress-ip/_index.md

```diff
@@ -83,13 +83,27 @@ oc get node -o json | \
 installed, you can review the `metadata.annotations['cloud.network.openshift.io/egress-ipconfig']` 
 field of each node manually to verify node capacity.
 
-### Create the Egress IP Rule
+### Create the Egress IP Rule(s)
 
 > **NOTE:** generally speaking it would be ideal to [label the nodes](#label-the-nodes) prior to assigning 
 the egress IP addresses, however there is a bug that exists which needs to 
 be fixed first.  Once this is fixed, the process and documentation will
 be re-ordered to address this.  See https://issues.redhat.com/browse/OCPBUGS-4969
 
+#### Identity the Egress IPs
```

Comment:
`Identity` -> `Identify`

Reply:
Fixed in 89ee7161ef1adb3067c25eaec742ebd089806719